io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,650 @@
+    bamm:operations ();
+    bamm:events ();
+    bamm:preferredName "Sealant Pass"@en;
+    bamm:description "The Sealant Passport represents the needs from manufacturers of chemical subtance and mixtures. It describes data required by the safety data sheet (SDS) - which is based on the REACH Regulation No 1907/2006 in the format of a DPP."@en;
+    bamm:see <https://osha.europa.eu/en/legislation/directives/regulation-ec-no-1907-2006-of-the-european-parliament-and-of-the-council>.
+:documents a bamm:Property;
+    bamm:preferredName "Documents"@en;
+    bamm:description "Set of documents containing the description of components."@en;
+    bamm:characteristic :Document.
+:performanceProperties a bamm:Property;
+    bamm:preferredName "Performance properties"@en;
+    bamm:description "Describes the performance and durability of a substance or mixture."@en;
+    bamm:characteristic :PerformanceCharacteristic.
+:serialNumber a bamm:Property;
+    bamm:preferredName "Serial number"@en;
+    bamm:description "Unique serial number to identify the material or batch."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "134X5643FC6557-23".
+:productIdentification a bamm:Property;
+    bamm:preferredName "Product identification"@en;
+    bamm:description "Identification of the substance/mixture and of the company/undertaking."@en;
+    bamm:characteristic :ProductIdentificationCharacteristic.
+:application a bamm:Property;
+    bamm:preferredName "Application"@en;
+    bamm:description "Relevant identified uses of the substance or mixture and uses advised against."@en;
+    bamm:characteristic :ApplicationCharacteristic.
+:supplier a bamm:Property;
+    bamm:preferredName "Supplier"@en;
+    bamm:description "Supplier of the mixture or substance."@en;
+    bamm:characteristic :SupplierCharacteristic.
+:hazardsIdentification a bamm:Property;
+    bamm:preferredName "Hazards identification"@en;
+    bamm:description "Describes the hazards of the substance or mixture and the appropriate warning information associated with those hazards."@en;
+    bamm:characteristic :HazardsIdentification.
+:productComposition a bamm:Property;
+    bamm:preferredName "Product composition"@en;
+    bamm:description "Information on the ingredients."@en;
+    bamm:characteristic :ProductComposition.
+:safetyMeasure a bamm:Property;
+    bamm:preferredName "Safety measure"@en;
+    bamm:description "Describes measures related to the safe handling of the mixtures and substances."@en;
+    bamm:characteristic :SafetyMeasureCharacteristic.
+:productProperties a bamm:Property;
+    bamm:preferredName "Product properties "@en;
+    bamm:description "Describes the physical and chemical properties of the substance or mixture."@en;
+    bamm:characteristic :ProductProperties.
+:productDisposal a bamm:Property;
+    bamm:preferredName "Product disposal"@en;
```

Comment:
you still have both versions with only first word, first letter uppercase and also all words, first sentence uppercase. Decide on one approach and be consistent. Please check all occurences